lib/tennis_set.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative 'tennis_game'
+require_relative 'tennis_score'
+
+# defines a tennis set, and handles all scoring for the game
+class TennisSet < TennisScore
+  def initialize(skip_games, players)
+    super(6, 2, 'set', players)
```

Comment:
Minor point - when the score is 5-5 the set lasts 7 games, but appreciate this may be out of scope for now.